backend/db_migrations/000019_duplicate-mealplan-name-input.up.sql

```diff
@@ -0,0 +1,27 @@
+comment on column app.meal_plan_entry.days is 'MONDAY: 0, TUESDAY: 1.., SUNDAY: 6';
+
+create or replace function app.duplicate_meal_plan(mealplan_id bigint, p_id bigint) returns app.meal_plan as $$
+declare
+m app.meal_plan;
+entry_ids bigint[];
+begin
+--create a duplicate meal plan with a different meal plan id and person id p_id but the same contents
+INSERT INTO app.meal_plan (name_en, name_fr, person_id, description_en, description_fr, tags)
+ SELECT name_en||' '||'copy', name_fr||' '||'copie', p_id as person_id, description_en, description_fr, tags FROM app.meal_plan WHERE id=mealplan_id
```

Comment:
If tomorrow we want to change this copy to say copy 1 or copy 2, then again you need to do a migration. So instead the best way to do it would be to get the meal plan name as a parameter. You can search for duplicateMealPlan function and the associated graphql query. You need to pass the mealplan name with the word copy as parameter in while calling the duplicateMealPlan function and modify the graphql query to pass on the mealplan name.